components/inputs/test/input-date.test.js

```diff
@@ -45,35 +45,87 @@ describe('d2l-input-date', () => {
 
 	});
 
-	describe('focus trap', () => {
```

Comment:
I moved these tests to just be combined with the `interacting with dropdown` tests since the setup was the same